geoserver_init/index.js

```diff
@@ -54,11 +55,31 @@ async function initGeoserver() {
 
   await createWorkspaces();
 
+  await setProxyBaseUrl();
```

Comment:
Please add this step to the list in the function API docs above (around line 49).

Reply:
done with f86b1bedff1fb5c2ceabdbd1e491ed659bb00e7b